delete lead to panic

```
「xx」 write successful!
+------------+-----------+-----------------------+--------------------+------------------------------+--------------+
|   CURRENT  |    NAME   |        CLUSTER        |        USER        |            SERVER            |   Namespace  |
+============+===========+=======================+====================+==============================+==============+
|            |   2123d   |   cluster-46gc56tgt8  |   user-46gc56tgt8  |   https://xxx  |    default   |
+------------+-----------+-----------------------+--------------------+------------------------------+--------------+
|            |    1234   |   cluster-46gc56tgt8  |   user-46gc56tgt8  |   https://xxx  |    default   |
+------------+-----------+-----------------------+--------------------+------------------------------+--------------+
|            |   xassss  |   cluster-46gc56tgt8  |   user-46gc56tgt8  |   https://xxx  |    default   |
+------------+-----------+-----------------------+--------------------+------------------------------+--------------+

❯ ./main delete
😸 Select:2123d
👻 True
Context Delete:「2123d」
「/Users/root/.kube/config」 write successful!
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x1eab718]

goroutine 1 [running]:
github.com/sunny0826/kubecm/cmd.PrintTable(0xc0004105a0, 0xc0001a8008, 0x212e220)
        /Users/root/projects/kubecm/cmd/utils.go:93 +0x178
github.com/sunny0826/kubecm/cmd.WriteConfig(0xc000219d01, 0xc000324c20, 0x17, 0xc0004105a0, 0x0, 0x0)
        /Users/root/projects/kubecm/cmd/utils.go:230 +0x16f
github.com/sunny0826/kubecm/cmd.(*DeleteCommand).runDelete(0xc0003bf558, 0xc000360dc0, 0x2b43b60, 0x0, 0x0, 0x0, 0x0)
        /Users/root/projects/kubecm/cmd/delete.go:56 +0x165
github.com/sunny0826/kubecm/cmd.(*DeleteCommand).Init.func1(0xc000360dc0, 0x2b43b60, 0x0, 0x0, 0x0, 0x0)
        /Users/root/projects/kubecm/cmd/delete.go:25 +0x52
github.com/spf13/cobra.(*Command).execute(0xc000360dc0, 0x2b43b60, 0x0, 0x0, 0xc000360dc0, 0x2b43b60)
        /Users/root/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:842 +0x47c
github.com/spf13/cobra.(*Command).ExecuteC(0xc0003602c0, 0x1046d17, 0x2aa00c0, 0xc000000180)
        /Users/root/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/root/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:887
main.main()
        /Users/root/projects/kubecm/main.go:27 +0x36

```

--- a/cmd/utils.go
+++ b/cmd/utils.go
@@ -90,8 +90,14 @@ func PrintTable(config *clientcmdapi.Config) error {
 		if obj.Namespace != "" {
 			namespace = obj.Namespace
 		}
-		server := config.Clusters[obj.Cluster].Server
-		conTmp := []string{head, key, obj.Cluster, obj.AuthInfo, server, namespace}
+		if config.Clusters == nil {
+			continue
+		}
+		cluster, ok := config.Clusters[obj.Cluster]
+		if !ok {
+			continue
+		}
+		conTmp := []string{head, key, obj.Cluster, obj.AuthInfo, cluster.Server, namespace}
 		table = append(table, conTmp)
 	}
 

